angular-client/src/pages/charging-page/components/battery-info-display/battery-info-display.ts

```diff
@@ -8,7 +8,7 @@ import Storage from 'src/services/storage.service';
   templateUrl: './battery-info-display.html',
   styleUrls: ['./battery-info-display.css']
 })
-export class BatteryInfoDisplay {
+export class BatteryInfoDisplayComponent implements OnInit {
```

Comment:
Why do some of these now implement OnInit (is it just convention through prettier for angular?)

Reply:
Yeah (it's actually the linter tho), it just makes the typing more explicit for the Components that implement the OnInit method, no actual functionality changes or anything